tests/rds.tf

```diff
@@ -20,14 +20,19 @@ module "rds_mysql_logs" {
 
 /*** RDS ***/
 
+
+data "aws_rds_engine_version" "rds_mysql" {
```

Comment:
Is this to deal with falling behind versions with automatic upgrades being done?